### ISSUE TITLE
Resource attribute "@pro" was not found

### ISSUE THREAD

**Issue by wiiznokes** [GitJournal/GitJournal#918]
I'm trying to build the project but i have this error:

```
Resource attribute "@pro" was not found. Please ensure that each resource has a corresponding @resource.
Generating synthetic localizations package has failed.
```
I followed the steps in build.md. I'm on Fedora, and i use version 1.22.6 of the flutter sdk.

**Comment by wiiznokes** [comment]
I have this new error now
```
flutter run --flavor dev --debug
Launching lib/main.dart on ONEPLUS A6013 in debug mode...
Checking the license for package NDK (Side by side) 21.4.7075529 in /home/lenaic/Android/Sdk/licenses
License for package NDK (Side by side) 21.4.7075529 accepted.
Preparing "Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)".
"Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)" ready.
Installing NDK (Side by side) 21.4.7075529 in /home/lenaic/Android/Sdk/ndk/21.4.7075529
"Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)" complete.
"Install NDK (Side by side) 21.4.7075529 (revision: 21.4.7075529)" finished.
Checking the license for package CMake 3.18.1 in /home/lenaic/Android/Sdk/licenses
License for package CMake 3.18.1 accepted.
Preparing "Install CMake 3.18.1 (revision: 3.18.1)".
"Install CMake 3.18.1 (revision: 3.18.1)" ready.
Installing CMake 3.18.1 in /home/lenaic/Android/Sdk/cmake/3.18.1
"Install CMake 3.18.1 (revision: 3.18.1)" complete.
"Install CMake 3.18.1 (revision: 3.18.1)" finished.
Checking the license for package Android SDK Tools in /home/lenaic/Android/Sdk/licenses
License for package Android SDK Tools accepted.
Preparing "Install Android SDK Tools (revision: 26.1.1)".
"Install Android SDK Tools (revision: 26.1.1)" ready.
Installing Android SDK Tools in /home/lenaic/Android/Sdk/tools
"Install Android SDK Tools (revision: 26.1.1)" complete.
"Install Android SDK Tools (revision: 26.1.1)" finished.
Checking the license for package Android SDK Build-Tools 30.0.3 in /home/lenaic/Android/Sdk/licenses
License for package Android SDK Build-Tools 30.0.3 accepted.
Preparing "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)".
"Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" ready.
Installing Android SDK Build-Tools 30.0.3 in /home/lenaic/Android/Sdk/build-tools/30.0.3
"Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" complete.
"Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" finished.
Checking the license for package Android SDK Platform 30 in /home/lenaic/Android/Sdk/licenses
License for package Android SDK Platform 30 accepted.
Preparing "Install Android SDK Platform 30 (revision: 3)".
"Install Android SDK Platform 30 (revision: 3)" ready.
Installing Android SDK Platform 30 in /home/lenaic/Android/Sdk/platforms/android-30
"Install Android SDK Platform 30 (revision: 3)" complete.
"Install Android SDK Platform 30 (revision: 3)" finished.
Checking the license for package Android SDK Platform 31 in /home/lenaic/Android/Sdk/licenses
License for package Android SDK Platform 31 accepted.
Preparing "Install Android SDK Platform 31 (revision: 1)".
"Install Android SDK Platform 31 (revision: 1)" ready.
Installing Android SDK Platform 31 in /home/lenaic/Android/Sdk/platforms/android-31
"Install Android SDK Platform 31 (revision: 1)" complete.
"Install Android SDK Platform 31 (revision: 1)" finished.
Checking the license for package Android SDK Platform 28 in /home/lenaic/Android/Sdk/licenses
License for package Android SDK Platform 28 accepted.
Preparing "Install Android SDK Platform 28 (revision: 6)".
"Install Android SDK Platform 28 (revision: 6)" ready.
Installing Android SDK Platform 28 in /home/lenaic/Android/Sdk/platforms/android-28
"Install Android SDK Platform 28 (revision: 6)" complete.
"Install Android SDK Platform 28 (revision: 6)" finished.
lib/generated/core.pb.dart:207:233: Error: No named parameter with the name 'valueDefaultOrMaker'.
    ..m<$core.String, Union>(6, _omitFieldNames ? '' : 'extraProps', protoName: 'extraProps', entryClassName: 'Note.ExtraPropsEntry', keyFieldType: $pb.PbFieldType.OS, valueFieldType: $pb.PbFieldType.OM, valueCreator: Union.create, valueDefaultOrMaker: Union.getDefault, packageName: const $pb.PackageName('gitjournal'))
                                                                                          ^^^^^^^^^^^^^^^^^^^
lib/generated/core.pb.dart:391:199: Error: No named parameter with the name 'valueDefaultOrMaker'.
    ..m<$core.String, Union>(2, _omitFieldNames ? '' : 'map', entryClassName: 'MdYamlDoc.MapEntry', keyFieldType: $pb.PbFieldType.OS, valueFieldType: $pb.PbFieldType.OM, valueCreator: Union.create, valueDefaultOrMaker: Union.getDefault, packageName: const $pb.PackageName('gitjournal'))
                                                                                          ^^^^^^^^^^^^^^^^^^^
lib/generated/core.pb.dart:491:228: Error: No named parameter with the name 'valueDefaultOrMaker'.
    ..m<$core.String, Union>(6, _omitFieldNames ? '' : 'mapValue', protoName: 'mapValue', entryClassName: 'Union.MapValueEntry', keyFieldType: $pb.PbFieldType.OS, valueFieldType: $pb.PbFieldType.OM, valueCreator: Union.create, valueDefaultOrMaker: Union.getDefault, packageName: const $pb.PackageName('gitjournal'))
                                                                                          ^^^^^^^^^^^^^^^^^^^
lib/generated/builders.pb.dart:46:220: Error: No named parameter with the name 'valueDefaultOrMaker'.
    ..m<$core.String, TzDateTime>(3, _omitFieldNames ? '' : 'map', entryClassName: 'BlobCTimeBuilderData.MapEntry', keyFieldType: $pb.PbFieldType.OS, valueFieldType: $pb.PbFieldType.OM, valueCreator: TzDateTime.create, valueDefaultOrMaker: TzDateTime.getDefault, packageName: const $pb.PackageName('gitjournal'))
                                                                                          ^^^^^^^^^^^^^^^^^^^
lib/generated/builders.pb.dart:115:226: Error: No named parameter with the name 'valueDefaultOrMaker'.
    ..m<$core.String, FileMTimeInfo>(3, _omitFieldNames ? '' : 'map', entryClassName: 'FileMTimeBuilderData.MapEntry', keyFieldType: $pb.PbFieldType.OS, valueFieldType: $pb.PbFieldType.OM, valueCreator: FileMTimeInfo.create, valueDefaultOrMaker: FileMTimeInfo.getDefault, packageName: const $pb.PackageName('gitjournal'))
                                                                                          ^^^^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script '/home/lenaic/Documents/flutter/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 1297

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDevDebug'.
> Process 'command '/home/lenaic/Documents/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7m 30s
Running Gradle task 'assembleDevDebug'...                         450,9s
Exception: Gradle task assembleDevDebug failed with exit code 1
```

**Comment by wiiznokes** [comment]
I finally manage to build this thing, a tried to generating the .proto files but there was already generated which is weird imo. 

This is how what i made:

- Download flutter sdk version 3.13.2: https://docs.flutter.dev/get-started/install/linux
- Add it to path: `PATH="$HOME/Documents/flutter/bin:$PATH"`
- Add platform-tools to path: `PATH="$HOME/Android/Sdk/platform-tools:$PATH"`. The sdk can be downloaded using Android studio. I also downloaded  the NDK (i have 25 and 21, idk the one flutter use rn). i also have Android SDK Build-Tools  33, CMake, Command line tools but idk if this is useful.
- run flutter doctor
- connect your device and run `flutter run --flavor dev --debug` (i think an helpful feature would be to specifie the device in this command)


For genering proto file (not needed)
- sudo dnf install reuse protoc
- change flutter format lib/ to dart format lib/ in Makefile
- `dart pub global activate protoc_plugin`
- `PATH="$HOME/.pub-cache/bin:$PATH"`
- make protos

**Comment by wiiznokes** [comment]
When i try to build for Linux, i get this error:
```
flutter run --flavor dev --debug
Launching lib/main.dart on Linux in debug mode...
Building Linux application...                                           

(gitjournal:350602): Gdk-CRITICAL **: 21:17:44.913: gdk_window_get_state: assertion 'GDK_IS_WINDOW (window)' failed
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method getSupportedModes on channel flutter_display_mode)
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:308:7)
<asynchronous suspension>
#1      FlutterDisplayMode.supported (package:flutter_displaymode/src/flutter_display_mode.dart:16:10)
<asynchronous suspension>
#2      FlutterDisplayMode.setHighRefreshRate (package:flutter_displaymode/src/flutter_display_mode.dart:58:37)
<asynchronous suspension>
#3      main (package:gitjournal/main.dart:35:3)
<asynchronous suspension>

Syncing files to device Linux...                                   207ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

A Dart VM Service on Linux is available at: http://127.0.0.1:34311/z7RhPAFu6kQ=/
The Flutter DevTools debugger and profiler on Linux is available at:
http://127.0.0.1:9100?uri=http://127.0.0.1:34311/z7RhPAFu6kQ=/
```
